docs/features/techdocs/how-to-guides.md

```diff
@@ -504,6 +506,18 @@ folder (/docs) or replace the content in this file.
 
 Done! You now have support for TechDocs in your own software template!
 
+### Prevent download of Google fonts
+
+If your Backstage instance does not have internet access, the generation will fail. TechDocs tries to download the Roboto font from Google. You can disable it by adding the following lines to mkdocs.yaml:
```

Comment:
We use a tool called Vale to spell check and confirm writing style. In this case it does not like the word "Roboto". You can fix this by adding it to this file - https://github.com/backstage/backstage/blob/master/.github/vale/Vocab/Backstage/accept.txt - and then including that in this PR

Reply:
I added "Roboto" in the accept.txt